main.pm

```diff
@@ -857,6 +857,14 @@ else {
     }
     elsif (get_var("BOOT_HDD_IMAGE")) {
         loadtest "boot/boot_to_desktop.pm";
+        if (get_var("ISCSI_SERVER")) {
+            set_var('INSTALLONLY', 1);
```

Comment:
`bmwqemu::vars` will be updated anyway so I think test will work. Only thing is that you won't see this variable set in webui, which will bite @dzedro in few months.
Or am I missing something?